src/Backoffice/Airlines/Domain/DataTransferObjects/AirlineDto.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airlines\Domain\DataTransferObjects;
+
+class AirlineDto
+{
+    public function __construct(
+        private readonly string $name,
+        private readonly string $description,
+    ) {
+    }
+
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    public function getDescription(): string
+    {
+        return $this->description;
+    }
```

Comment:
```suggestion
readonly class AirlineDto
{
    public function __construct(
        public string $name,
        public string $description,
    ) {
    }
```
While getters are what you're supposed to use to prevent breaking encapsulation, by having readonly properties, you don't need them and can access properties directly.
```
$dto = new AirlineDto('Turkish Airlines', 'An airline from Turkey');

$dto->name;
$dto->description;
```